salt/metalk8s/salt/master/files/master-99-metalk8s.conf.j2

```diff
@@ -59,3 +59,9 @@ kubernetes.kubeconfig: {{ kubeconfig }}
 {%- if kubecontext is defined and kubecontext %}
 kubernetes.context: {{ kubecontext }}
 {%- endif %}
+
+reactor:
+  - 'salt/beacon/*/watch_certificates_expiry/':
```

Comment:
Should we namespace our event tags? CC @gdemonet

Reply:
We can't namespace beacon events, I believe. IIUC, the format will always be:

```
salt/beacon/$minion_id/$beacon_config_name/$tag
```

Reply:
If we really want to, we can add a reactor to transform event tags, but feels unnecessary.

Reply:
We could kind of namespace our events, just pre-prending beacon config name with metalk8s:
```
beacons:
  metalk8s_watch_certificates_expiry:
  - files:
    - /etc/metalk8s/pki/dex/server.crt
    [...]
  - notify_days: 45
  - disable_during_state_run: true
  - interval: 86400
  - beacon_module: cert_info
```

Reply:
Oh yeah, I meant they'll always live under `salt/beacon/$minion_id`. Using prefixed names can be useful if we expect other beacons to be configured, good idea.